src/style.scss

```diff
@@ -276,65 +276,65 @@ slot {
   }
 
   &Ondark {
-    background-color: var(--auro-color-ui-default-on-dark);
-    border: 1px solid var(--auro-color-ui-default-on-dark);
-    color: var(--auro-color-text-primary-on-light);
+    background-color: var(--ds-color-ui-default-inverse, $ds-color-ui-default-inverse);
+    border: 1px solid var(--ds-color-ui-default-inverse, $ds-color-ui-default-inverse);
+    color: var(--ds-color-text-primary-default);
 
     // auro-buttonOndark hover
     @media (hover: hover) {
       &:active,
       &:hover {
-        background-color: var(--auro-color-ui-hover-on-dark);
-        border: 1px solid var(--auro-color-ui-hover-on-dark);
+        background-image: linear-gradient(var(--ds-color-ui-hover-inverse, $ds-color-ui-hover-inverse), var(--ds-color-ui-hover-inverse, $ds-color-ui-hover-inverse));
+        border: 1px solid var(--ds-color-ui-hover-inverse, $ds-color-ui-hover-inverse);
       }
     }
 
     &:active {
-      background-color: var(--auro-color-ui-active-on-dark);
-      border: 1px solid var(--auro-color-ui-active-on-dark);
+      background-color: var(--ds-color-brand-breeze-200, $ds-color-brand-breeze-200);
+      border: 1px solid var(--ds-color-brand-breeze-200, $ds-color-brand-breeze-200);
```

Comment:
I'm not seeing in the tickets linked to change to the brand breeze color.

Reply:
@leeejune after some conversation with @jordanjones243 he switched to breeze because the original token here was disabled but the brand breeze color had the same hex code. 

Can you update us here what the correct token for the figma design should be under the new DS tokens?

Reply:
I've added a few missing design tokens in this issue. This should solve this conflict. https://github.com/AlaskaAirlines/AuroDesignTokens/issues/137